watchdog/src/lib.rs

```diff
@@ -74,6 +85,12 @@ pub fn get_config() -> Config {
     crate::storage::get_config()
 }
 
+/// Returns the API access of the Bitcoin canister.
```

Comment:
```suggestion
/// Returns the API access of the Bitcoin canister as known by the watchdog.
```

basically make it clear that this is the local information about the api access which might or might not be in sync.

Reply:
Not relevant any more after removing the `ApiAccess` struct. Now it's only storing the API access target value.